winlogbeat/docs/modules.asciidoc

```diff
@@ -0,0 +1,57 @@
+[id="{beatname_lc}-modules"]
+[role="xpack"]
+= Modules
+
+[partintro]
+--
+This section contains detailed information about the available Windows event
+log processing modules contained in {beatname_uc}. More details about each
+module can be found in the module's documentation.
+
+Winlogbeat modules are constructed using the
+<<processor-script,script processor>> where all processing happens within
+Winlogbeat before the events are delivered to the output.
+
+The general goal of each module is to transform events by renaming fields to
+comply with the Elastic Common Schema (ECS). The modules may also apply
```

Comment:
Maybe link to ECS docs (if somebody doesn't know what it is)?

Reply:
Added.